src/utils/constants.ts

```diff
@@ -31,3 +31,10 @@ export const contentTemplate = (generatedContent: string, originalContent: strin
   ${CONTENT_MARK}\n
 `;
 };
+
+export const rulesBlockTemplate = (source: string, entry: string[]): string => {
+  return stripIndents`
+  # Rule extracted from ${source}
```

Comment:
```suggestion
  # Rules extracted from ${source}
```

Reply:
I think this needs some logic. I will make it happen